qiskit/circuit/bit.py

```diff
@@ -70,14 +79,23 @@ def register(self):
 
     @property
     def index(self):
-        """Get bit's index."""
+        """Get the index of an old-style bit in the register that owns it.
+
+        .. deprecated:: Qiskit Terra 0.17
```

Comment:
Same as https://github.com/Qiskit/qiskit-terra/pull/9706/files#r1123193630
```suggestion
        .. deprecated:: 0.17
```